client/my-sites/themes/theme-download-card/index.jsx

```diff
@@ -21,7 +21,7 @@ var ThemeDownloadCard = React.createClass( {
 	},
 
 	render() {
-		var downloadURI = this.props.href || ( 'http://wordpress.org/themes/' + this.props.theme );
+		var downloadURI = this.props.href || ( 'https://downloads.wordpress.org/theme/' + this.props.theme + '.zip' );
```

Comment:
Does this mean that if it's empty we assume it's on .org? Is this a fair assumption to make?